TIGLViewer/src/StandardizeDialog.cpp

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (C) 2019 CFS Engineering
+ *
+ * Created: 2019 Malo Drougard <malo.drougard@protonmail.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "StandardizeDialog.h"
+#include "ui_StandardizeDialog.h"
+#include "CCPACSWing.h"
+#include "CCPACSFuselage.h"
+#include "CTiglStandardizer.h"
+#include "QRegExp"
+#include "QRegularExpressionMatch"
```

Comment:
This seems to be unused. Can we remove it? It requires Qt 5.0 or higher

Reply:
Yes :)

Reply:
Done